### ISSUE TITLE
Proxy-Problems

### ISSUE THREAD

**Issue by Telvanis** [avatsaev/angular4-docker-example#7]
Hello there.
I've got a problem building the image. npm seems to reach some IPs but isnt able to connect.

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: aproba@1.1.2 (node_modules/aproba):
npm WARN network SKIPPING OPTIONAL DEPENDENCY: request to https://registry.npmjs.org/aproba/-/aproba-1.1.2.tgz failed, reason: connect ETIMEDOUT 151.101.112.162:443

I am behind a proxy. How can I change your Dockerfile to setup the proxy for npm?
Would it be enough to just use this command?
`npm config set proxy http://<username>:<password>@<proxy-server-url>:<port>`

I would insert it directly before or after RUN `npm set progress=false && npm config set depth 0 && npm cache clean --force `

Do you have a solution? :/

Thank you in advance :)

**Comment by avatsaev** [comment]
Try creating and copying an npmrc config file into the image https://gist.github.com/ikaruce/aaaf884a1f48555b5d5f

**Comment by Telvanis** [comment]
Ok, I tried your thought but either I did it wrong or it rly does not work :/ I created a new file inside the angular4-docker-example folder and inserted the comments. But I received the same error message.

So I tried my own solution; I inserted `npm config set proxy` as step 3 and everything worked fine. 
Thank you for the image, Angular seems to run like it has to do... Now I just have to test some apps, let's see, how I implement them ^^

Sincerely Telvanis

PS: Here is the turtorial I followed, just for ppl who wanna try it too 
[https://oliverveits.wordpress.com/2017/10/02/angular-4-docker-example/](url)
